src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -6603,13 +6880,51 @@ public void ParameterScope_07()
 @"ref struct R { }
 class Program
 {
-    static void F1(scoped scoped R r) { }
-    static void F2(ref scoped scoped R r) { }
+    static void F0(scoped scoped R r) { }
+    static void F1(ref scoped scoped R r) { }
+    static void F2(scoped ref scoped R r) { }
     static void F3(scoped scoped ref R r) { }
+    static void F4(in scoped scoped R r) { }
+    static void F5(scoped in scoped R r) { }
+    static void F6(scoped scoped in R r) { }
+    static void F7(out scoped scoped R r) { r = default; }
+    static void F8(scoped out scoped R r) { r = default; }
+    static void F9(scoped scoped out R r) { r = default; }
 }";
             var comp = CreateCompilation(source);
-            // PROTOTYPE: Should report duplicate modifiers.
-            comp.VerifyEmitDiagnostics();
+            comp.VerifyEmitDiagnostics(
+                // (4,27): error CS1107: A parameter can only have one 'scoped' modifier
```

Comment:
I found this slightly surprising. I thought we were going to make 'scoped' as a type name an error like we did for 'required' as a type name.

Reply:
We're not reporting an error for `scoped` as a  type name yet.